frontend/front/src/api/apiSlice.js

```diff
@@ -1,12 +1,23 @@
 import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
 
+import { AUTHORIZATION_HEADER } from "../features/login/loginUtils";
+
+const baseUrl = process.env.REACT_APP_API_URL || "http://localhost:3001";
+
 const sortById = (a, b) => a.id - b.id;
 
 export const apiSlice = createApi({
   reducerPath: "apiSlice",
   baseQuery: fetchBaseQuery({
-    // baseUrl: "http://localhost:3001",
-    baseUrl: "https://testing-ibhy.onrender.com",
+    baseUrl,
+    prepareHeaders: (headers, { getState }) => {
+      const token = getState().login.token;
+      // add auth to every request, if the token exists
+      if (token) {
+        headers.set(AUTHORIZATION_HEADER, token);
```

Comment:
This is the important part, this adds the `authorization` header to every request, if we have it.